packages/core/src/schema.ts

```diff
@@ -32,8 +32,9 @@ const _schema = [
                 type: 'Int',
                 default: () => +new Date(),
             },
-            ['postId', 'String'],
-            ['commentId', 'String', { optional: true }],
+            ['postId', 'String', { optional: true }],
+            ['onChainPostId', 'String'],
+            ['onChainCommentId', 'String', { optional: true }],
```

Comment:
This can just be `onChainId`. If we're referencing this on a comment it's like `comment.onChainId` instead of `comment.onChainCommentId`.